### ISSUE TITLE
StackOverflowError in IDEA 2018.1.2 EAP

### ISSUE THREAD

**Issue by chrkv** [reasonml-editor/reasonml-idea-plugin#45]
IDEA 181.4668

```
<< Worker exited due to exception

java.lang.StackOverflowError
	at com.intellij.openapi.progress.impl.CoreProgressManager.doCheckCanceled(CoreProgressManager.java:120)
	at com.intellij.openapi.progress.ProgressManager.checkCanceled(ProgressManager.java:208)
	at com.intellij.openapi.progress.ProgressIndicatorProvider.checkCanceled(ProgressIndicatorProvider.java:49)
	at com.intellij.psi.impl.source.tree.CompositeElement.getPsi(CompositeElement.java:727)
	at com.intellij.psi.impl.source.SourceTreeToPsiMap.treeElementToPsi(SourceTreeToPsiMap.java:30)
	at com.intellij.psi.impl.source.tree.SharedImplUtil.getParent(SharedImplUtil.java:47)
	at com.intellij.extapi.psi.ASTWrapperPsiElement.getParent(ASTWrapperPsiElement.java:36)
	at com.intellij.psi.impl.PsiElementBase.isValid(PsiElementBase.java:254)
	at org.jetbrains.plugins.ruby.i18n.locales.I18nRubyAdapter.lambda$isTranslationElement$0(I18nRubyAdapter.java:39)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:927)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:47)
	at org.jetbrains.plugins.ruby.i18n.locales.I18nRubyAdapter.isTranslationElement(I18nRubyAdapter.java:38)
	at org.jetbrains.plugins.ruby.i18n.findUsages.I18nElementDescriptionProvider.getElementDescription(I18nElementDescriptionProvider.java:23)
	at com.intellij.psi.ElementDescriptionUtil.getElementDescription(ElementDescriptionUtil.java:30)
	at com.reason.ide.search.OclFindUsagesProvider.getType(OclFindUsagesProvider.java:41)
	at com.intellij.usageView.UsageViewTypeLocation$1.getElementDescription(UsageViewTypeLocation.java:67)
...
	at com.intellij.psi.ElementDescriptionUtil.getElementDescription(ElementDescriptionUtil.java:38)
	at com.reason.ide.search.OclFindUsagesProvider.getType(OclFindUsagesProvider.java:41)
	at com.intellij.usageView.UsageViewTypeLocation$1.getElementDescription(UsageViewTypeLocation.java:67)
...
    300 times
...
```

**Comment by jsiebern** [comment]
Getting the same sometimes (usually after longer running times of the IDE)

**Comment by giraud** [comment]
I consider it to be fixed in 0.47 because these classes have been deleted
